src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.db.DataBase;
+import java.util.List;
+
+public class ConverterDataToObjImpl implements ConverterDataToObj {
+    private static final String SPLITERATOR = ",";
+    private static final int OPERATION_SEQUENCE = 0;
+    private static final int FRUIT_NAME_SEQUENCE = 1;
+    private static final int FRUIT_QUANTITY_SEQUENCE = 2;
+
+    @Override
+    public void convertData(String filePath) {
+        ReaderDataFromFileImpl readerDataFromFile = new ReaderDataFromFileImpl();
+
+        List<FruitTransaction> list = readerDataFromFile.readFromFile(filePath).stream()
+                .map(s -> s.split(SPLITERATOR))
```

Comment:
avoid short one-letter naming even in stream chains, readability is a pririority